files/default/lib/bumpzone.rb

```diff
@@ -0,0 +1,112 @@
+#!/opt/chef/embedded/bin/ruby
+require 'git'
+require 'json'
+require 'octokit'
+
+# Library to bump DNS zones
+class BumpZone
+  @commit_msg = "\n\n"
+  @warning_msg = "Warning:\n\n"
+  @warning = false
+  ONELINE_REGEX = /(^[\w\d\.\-]+\s+[\d]+\s+IN\s+SOA.*[\w\d\.\-]+\s+)(\d{10})(\s+[\d]+\s+[\d]+\s+[\d]+\s+[\d]+)/
+  MULTILINE_REGEX =
+    /(^[\w\d\.\-]+\s+IN\s+SOA\s+[\w\d\.\-]+\s+[\w\d\.\-]+\s+\(\s?\n\s+)(\d{10})(.*\n.*\n.*\n.*\n.*\n\s+\))/
+
+  class << self
+    attr_reader :commit_msg
+  end
+
+  def self.add_commit_msg(msg)
+    @commit_msg << msg
```

Comment:
I format them so that they are readable. What would you suggest I do differently?